docfiles/footer.html

```diff
@@ -22,6 +22,9 @@
             <a class="item" href="https://makecode.com/privacy" target="_blank" rel="noopener">Privacy &amp; Cookies</a>
             <a class="item" href="https://makecode.com/termsofuse" target="_blank" rel="noopener"> Terms Of Use</a>
             <a class="item" href="https://makecode.com/trademarks" target="_blank" rel="noopener">Trademarks</a>
+            <a id="langpicker" class="ui item" rel="noopener">
```

Comment:
```suggestion
            <a id="langpicker" class="ui item" rel="noopener" tabindex="0">
```

without an ``href`` this isn't in tab order by default